enhancements/storage/csi-driver-operator-merge.md

```diff
@@ -0,0 +1,444 @@
+| [openshift/gcp-filestore-csi-driver-operator](https://github.com/openshift/gcp-filestore-csi-driver-operator/)         | ose-gcp-filestore-csi-driver-operator.yml    |
+| [openshift/ibm-powervs-block-csi-driver-operator](https://github.com/openshift/ibm-powervs-block-csi-driver-operator/) | ose-powervs-block-csi-driver-operator.yml    |
+| [openshift/ibm-vpc-block-csi-driver-operator](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/)         | ose-ibm-vpc-block-csi-driver-operator.yml    |
+| [openshift/openstack-cinder-csi-driver-operator](https://github.com/openshift/openstack-cinder-csi-driver-operator/)   | ose-openstack-cinder-csi-driver-operator.yml |
+| [openshift/csi-driver-manila-operator](https://github.com/openshift/csi-driver-manila-operator/)                       | csi-driver-manila-operator.yml               |
+| [openshift/ovirt-csi-driver-operator](https://github.com/openshift/ovirt-csi-driver-operator/)                         | ose-cluster-ovirt-csi-operator.yml           |
+| [openshift/secrets-store-csi-driver-operator](https://github.com/openshift/secrets-store-csi-driver-operator/)         | ose-secrets-store-csi-driver-operator.yml    |
+| [openshift/csi-driver-shared-resource-operator](https://github.com/openshift/csi-driver-shared-resource-operator/)     | ose-csi-driver-shared-resource-operator.yml  |
+| [openshift/vmware-vsphere-csi-driver-operator](https://github.com/openshift/vmware-vsphere-csi-driver-operator/)       | ose-vmware-vsphere-csi-driver-operator.yml |
+
+*) ART name is the name of metadata file in https://github.com/openshift-eng/ocp-build-data/tree/openshift-4.14/images
+
+During development of an OCP release, we will merge an CSI driver operator to github.com/openshift/csi-operator
+without any testing in CI. We will keep as close to the original operator as possible, and we will test the new image
+manually (or with some tricks in CI if possible, see below).
+
+Once we're happy with the results, we will follow ART's guidelines and flip building of an existing image from the old
+repo to github.com/openshift/csi-operator. The resulting image name will be the same. This will require a tight
+cooperation with ART. We expect guidelines
+like [Changing the component name of a second level operator](https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#changing-the-component-name-of-a-second-level-operator) -
+we're not changing name of an operator, but we want to change the source repository of the image.
+
+Example:
+
+* In OCP 4.14, ART builds
+  only [`ose-aws-ebs-csi-driver-operator-container`](https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=74505)
+  from https://github.com/openshift/aws-ebs-csi-driver-operator repository.
+
+* In 4.15, we merge the operator into github.com/openshift/csi-operator and do not change any code of it,
+  except for directory names. This way, we can ensure the code will be 99% the same as the old operator.
```

Comment:
Add more details about the directory structure and how will we refactor the old operator to the new common code.